test/testdata/config3

```diff
@@ -0,0 +1,24 @@
+# config with two contexts and a selected default
```

Comment:
Do we need a new file?
You can use another 2-context file, and set a context in the test (if that’s the only difference).

Reply:
Yeah good question, I didn't want to make existing / future tests more complicated by overloading fixtures but happy to merge into `config2` if you are.

Reply:
What I meant was that you can use an existing testdata file, and first set a context in test code, and then unset it. This way you wouldn't be modifying the existing testdata file.

Reply:
Ah that makes more sense!